src/deprecated.js

```diff
@@ -22,7 +22,8 @@ jQuery.fn.extend( {
 		return arguments.length === 1 ?
 			this.off( selector, "**" ) :
 			this.off( types, selector || "**", fn );
-	}
+	},
+	isArray: Array.isArray
```

Comment:
Nice catch. @kumarmj I know this is now fixed but have you run the test suite before submitting the PR? It would fail the test you added in the previous version.